File: benchmarkDS/Vanilla/OPENAI_gpt-5-2025-08-07/Lse75.java
public class Lse75 {
    static class Person {
        public int status;
        public Person(int status) { this.status = status; }
        @Override
        public String toString() { return "Person{status=" + status + "}"; }
    }

    // Allocates a new Person and sets status to 0
    public static Person allocatePerson() {
        return new Person(0);
    }

    public static void main(String[] args) {
        // 5 test cases
        for (int i = 0; i < 5; i++) {
            Person p = allocatePerson();
            System.out.println(p);
        }
    }
}

File: benchmarkDS/Vanilla/OPENAI_gpt-5-2025-08-07/Lse75.cpp
#include <iostream>
#include <cstdlib>

struct Person {
    int status;
};

// Allocates a block of memory of size sizeof(Person) and returns pointer to it.
// Sets status to 0.
Person* allocate_person() {
    Person* p = static_cast<Person*>(std::malloc(sizeof(Person)));
    if (p) {
        p->status = 0;
    }
    return p;
}

int main() {
    // 5 test cases
    for (int i = 0; i < 5; ++i) {
        Person* p = allocate_person();
        if (p) {
            std::cout << "Person{status=" << p->status << "}" << std::endl;
            std::free(p);
        } else {
            std::cout << "Allocation failed" << std::endl;
        }
    }
    return 0;
}